projects/plugins/jetpack/modules/subscriptions/subscribe-overlay/subscribe-overlay.css

```diff
@@ -24,13 +24,20 @@ body.jetpack-subscribe-overlay-open {
 	visibility: hidden;
 	overflow: hidden;
 	top: 100%;
-	margin: 15% auto;
+	/*margin: 15% auto;*/
 	width: 100%;
-	max-width: 400px;
+	/*max-width: 400px;*/
 	transition: top 0.4s, visibility 0.4s;
 	text-wrap: pretty;
 }
 
+/* These should be coming from the editor :-( */
+.jetpack-subscribe-overlay__content .wp-block-group.is-layout-flex {
```

Comment:
If we keep it (still prefer not), we need to make sure it's applied only to top-most group, perhaps by giving the group block a custom CSS class.

It's good to have `jetpack-subscribe-overlay__content` so that everything still works even if they remove the group block in the template accidentally.